chanreg.md

```diff
@@ -215,10 +215,12 @@ Registered communities can profit from the following benefits:
 The Projects & Community Team comprises, among others, e, Fuchs, glguy and
```

Comment:
I think this can be reworked to point at https://libera.chat/about#projects-and-community-team which would mean one less place to keep in sync. It would probably require a rewording of the entire section though

Reply:
I'll take a look but might not have time for it.

Reply:
I wonder if this could be better handled by having a page per staffer, with their nick, realname, and roles as tags, and leave Jekyll to build the lists within that page?